Cannot  build master, Docs, asciidoctor:, datasource-guide.adoc: line 228: include file not found

Build fails on Documentation build: 
```
./mvnw install

[INFO] --- asciidoctor-maven-plugin:2.0.0-RC.1:process-asciidoc (output-html) @ quarkus-documentation ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 17 resources
[INFO] Rendered /home/karm/workspaceRH/quarkus/docs/src/main/asciidoc/getting-started-knative-guide.adoc
[INFO] asciidoctor: ERROR: datasource-guide.adoc: line 228: include file not found: /home/karm/workspaceRH/quarkus/docs/src/main/asciidoc/generated/quarkus-agroal.adoc
[INFO] Rendered /home/karm/workspaceRH/quarkus/docs/src/main/asciidoc/datasource-guide.adoc
[ERROR] asciidoctor: ERROR: datasource-guide.adoc: line 228: include file not found: /home/karm/workspaceRH/quarkus/docs/src/main/asciidoc/generated/quarkus-agroal.adoc
[INFO] ------------------------------------------------------------------------

```
**Environment:**
Linux, Java 11, Quarkus git HEAD: 7404e8e

--- a/docs/pom.xml
+++ b/docs/pom.xml
@@ -38,7 +38,7 @@
                     <enableVerbose>true</enableVerbose>
                     <logHandler>
                         <failIf>
-                          <severity>INFO</severity>
+                          <severity>WARN</severity>
                         </failIf>
                     </logHandler>
                     <attributes>

--- a/docs/src/main/asciidoc/datasource-guide.adoc
+++ b/docs/src/main/asciidoc/datasource-guide.adoc
@@ -109,6 +109,9 @@ quarkus.datasource.max-size=13
 
 There are other configuration options, detailed below.
 
+[TIP]
+For more information about the Agroal extension configuration please refer to the <<configuration-reference, Configuration Reference>>.
+
 === JDBC URL configurations
 
 Each of the supported databases contains different JDBC URL configuration options.
@@ -223,7 +226,9 @@ AgroalDataSource dataSource1;
 AgroalDataSource dataSource2;
 --
 
+
+[[configuration-reference]]
 == Agroal Configuration Reference
 
-include::generated/quarkus-agroal.adoc[]
+include::generated/quarkus-agroal.adoc[opts=optional, leveloffset=+1]
 

--- a/docs/src/main/asciidoc/hibernate-search-guide.adoc
+++ b/docs/src/main/asciidoc/hibernate-search-guide.adoc
@@ -544,8 +544,8 @@ While, on a production setup, the `committed` default might be a suitable value,
 This is for testing purposes with the Elasticsearch Docker container.
 It should not be used in production.
 
-== Hibernate Search Configuration
-include::generated/quarkus-hibernate-search-elasticsearch.adoc[]
+[TIP]
+For more information about the Hibernate Search extension configuration please refer to the <<configuration-reference, Configuration Reference>>.
 
 == Creating a frontend
 
@@ -661,3 +661,7 @@ Thus we focused our efforts on it.
 
 We don't have plans to support the Lucene backend in Quarkus for now.
 
+[[configuration-reference]]
+== Hibernate Search Configuration Reference
+
+include::generated/quarkus-hibernate-search-elasticsearch.adoc[leveloffset=+1, opts=optional]

--- a/docs/src/main/asciidoc/sending-emails.adoc
+++ b/docs/src/main/asciidoc/sending-emails.adoc
@@ -61,9 +61,8 @@ quarkus.mailer.username=....
 quarkus.mailer.password=....
 ----
 
-The following table lists the properties that can be configured:
-
-include::generated/quarkus-mailer-impl.adoc[]
+[TIP]
+For more information about the Mailer extension configuration please refer to the <<configuration-reference, Configuration Reference>>.
 
 == Sending simple emails
 
@@ -283,3 +282,9 @@ You can also create your own instance, and pass your own configuration.
 
 This guide has shown how you can send emails from a Quarkus application.
 The _mailer_ extension works in JVM and native mode.
+
+
+[[configuration-reference]]
+== Mailer Configuration Reference
+
+include::generated/quarkus-mailer-impl.adoc[opts=optional, leveloffset=+1]

